libcob/ChangeLog

```diff
@@ -1,4 +1,10 @@
 
+2025-11-09	Oğuzcan Kırmemiş <oguzcan.kirmemis@gmail.com>
+
+	* common.c: remove strbuff variable and cob_strcat/cob_strjoin utilities
+	which were prone to memory corruption due to strbuff being a global
+	* coblocal.h: remove definitions of cob_strcat and cob_strjoin
```

Comment:
An entry for `call.c (add_to_preload)` is missing

Reply:
added